common/views/components/CheckboxRadio/README.md

```diff
@@ -0,0 +1,4 @@
+## Purpose
+Render a styled checkbox or radio with a label.
```

Comment:
```suggestion
Render a styled radio with a label.
```

Reply:
The component can render either a checkbox or radio, depending on the `type` prop it receives.